examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -1512,22 +1582,37 @@ def compute_text_embeddings(prompt, text_encoders, tokenizers):
 
                 # Convert images to latent space
                 model_input = vae.encode(pixel_values).latent_dist.sample()
-                model_input = model_input * vae.config.scaling_factor
-                if args.pretrained_vae_model_name_or_path is None:
-                    model_input = model_input.to(weight_dtype)
+
+                if "playgroundai" not in args.pretrained_model_name_or_path:
```

Comment:
same comment as above. We should check of the `mean` and `std` args exist and not `None` instead of checking for `playgroundai`